superset-frontend/src/SqlLab/components/ExploreResultsButton/index.jsx

```diff
@@ -129,8 +119,8 @@ class ExploreResultsButton extends React.PureComponent {
     );
   }
 
-  renderInvalidColumnMessage() {
-    const invalidColumns = this.getInvalidColumns();
+  function renderInvalidColumnMessage() {
```

Comment:
Please convert this to an arrow function.

Also - GH actions says that this function is defined but never used. If you come across any unused functions like this, please remove them.